synapseclient/core/credentials/cached_sessions.py

```diff
@@ -75,7 +49,7 @@ def migrate_old_session_file_credentials_if_necessary(syn):
             if key == "<mostRecent>":
                 set_most_recent_user(value)
             else:
-                set_api_key(key, value)
+                SynapseApiKeyCredentials(key, value).store_to_keyring()
```

Comment:
we can potentially remove this old migration code now. This was implemented around 2-3 years ago for a 1-time migration of older client versions.